LayoutTests/css3/scroll-snap/scroll-snap-drag-scrollbar-thumb-with-sticky.html

```diff
@@ -73,6 +73,8 @@
                 eventSender.mouseMoveTo(20, 190);
                 eventSender.mouseDown();
                 eventSender.mouseMoveTo(100, 190);
+
+                await UIHelper.ensurePresentationUpdate();
```

Comment:
ah, I was trying to fix this test by using `renderingUpdate`, not `ensurePresentationUpdate`. What's the difference between the two?

Reply:
`renderingUpdate()` only waits for the next `requestAnimationFrame` (which happens when updating rendering), and waits for a 0-delay timer after that in the web process.

`ensurePresentationUpdate` waits:
1. ...for the UI process to trigger a rendering update in the web process
2. ...for the web process to handle the rendering update and send a layer tree commit back to the UI process
3. ...and finally, for the UI process to acknowledge the incoming layer tree commit.